CardPayments/src/main/java/com/paypal/android/cardpayments/api/DataVaultPaymentMethodTokensAPI.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.cardpayments.api
+
+import com.paypal.android.cardpayments.VaultRequest
+import com.paypal.android.cardpayments.VaultResult
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.RestClient
+
+// TODO: remove suppress statement when implementation is done
```

Comment:
Agree with @scannillo statement, we should keep this feature internal as a whole until it's ready